src/runtimes/node/parser/exports.ts

```diff
@@ -93,35 +124,41 @@ const getMainExportFromESM = (node: Statement, getAllBindings: BindingMethod) =>
   const handlerDeclaration = declaration.declarations.find((childDeclaration) => {
     const { id, type } = childDeclaration
 
-    return type === 'VariableDeclarator' && id.type === 'Identifier' && id.name === 'handler'
+    return type === 'VariableDeclarator' && id.type === 'Identifier' && id.name === name
   })
 
   const exports = getExportsFromExpression(handlerDeclaration?.init)
 
   return exports
 }
 
-// Check if the Node is an ExportSpecifier that has a named export called `handler`
-// either with Identifier `export { handler }`
-// or with StringLiteral `export { x as "handler" }`
-const isHandlerExport = (node: ExportNamedDeclaration['specifiers'][number]): node is ExportSpecifier => {
+/**
+ * Check if the node is an `ExportSpecifier` that has a named export with
+ * the given name, either as:
+ * - `export { handler }`, or
+ * - `export { x as "handler" }`
+ */
+const isNamedExport = (node: ExportNamedDeclaration['specifiers'][number], name: string): node is ExportSpecifier => {
```

Comment:
Renamed from `isHandlerExport`. This is now generic so it can look for exports with different names.